--- a/sentry-native.cmake	2025-04-17 22:32:30.000000000 +0800
+++ b/sentry-native.cmake	2025-08-12 22:39:43.220241470 +0800
@@ -15,9 +15,7 @@
 include(FetchContent)
 FetchContent_Declare(
     sentry-native
-    GIT_REPOSITORY ${SENTRY_NATIVE_repo}
-    GIT_TAG ${SENTRY_NATIVE_version}
-    EXCLUDE_FROM_ALL
+    SOURCE_DIR "/run/build/chat/sentry-native"
 )
 
 FetchContent_MakeAvailable(sentry-native)
